assets/helpers/contributions.js

```diff
@@ -122,3 +130,25 @@ export function errorMessage(
   }
 
 }
+
+function contribCamelCase(contrib: Contrib) {
```

Comment:
Since there are no tests for this file, you can create it and start adding tests at least for this function.

Reply:
Done.